dexter/src/main/java/com/karumi/dexter/Dexter.java

```diff
@@ -33,10 +33,12 @@
   private static DexterInstance instance;
 
   /**
-   * Initializes the library
+   * Initializes the library.
    *
    * @param context Context used by Dexter. Use your {@link android.app.Application} to make sure
-   * the instance is not cleaned up during your app lifetime
+   * the instance is not cleaned up during your app lifetime.
+   * @deprecated use the initialize overloaded method using an {@link android.app.Activity} instance
```

Comment:
The method should be annotated with the `@Deprecated` tag as well, AS is smart enough but other IDEs might not recognize the javadoc